### PR TITLE
Add Deprecated to Vanilla Waypoints

### PR DESCRIPTION
**Pull request by jokoho48**
### When merged, this pull request will:

1. *Title*

### CODE REVIEW

**Comment by diwako** [comment]
Needs info in which version they are removed.

**Comment by jokoho48** [comment]
> Needs info in which version they are removed.

We can't remove them without breaking Missions, so I think we may just mark them as "deprecated" and leave them.

File: addons/wp/stringtable.xml [hunk 1]
```diff
@@ -1221,5 +1221,59 @@
             <Italian>Registri artiglieria per uso dinamico</Italian>
             <Chinesesimp>动态使用方面的火炮指示</Chinesesimp>
         </Key>
+        <Key ID="STR_Lambs_WP_Waypoint_Deprecated_TaskAssault_displayName">
+            <English>TASK ASSAULT</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Waypoint_Deprecated_TaskAssault_Tooltip">
+            <English>Rushes to target position with no regard for safety\nDEPRECATED, USE MODULES FOR MORE FLEXIBILITY FOR MORE FLEXIBILITY</English>
```
Comment (johnb432) [inline addons/wp/stringtable.xml:1228]: `USE MODULES FOR MORE FLEXIBILITY FOR MORE FLEXIBILITY` - looks like a duplicate snuck in there.

Comment (jokoho48) [reply addons/wp/stringtable.xml:1228]: i blame @nk3nny

**Pull request merged**